newscoop/include/smarty/campsite_plugins/block.image.php

```diff
@@ -60,11 +60,17 @@ function smarty_block_image(array $params, $content, Smarty_Internal_Template $s
     }
 
     $imageService = \Zend_Registry::get('container')->getService('image');
+    $preferencesService = \Zend_Registry::get('container')->getService('preferences');
+    $caption = $imageService->getCaption($articleRendition->getImage(), $article->number, $article->language->number);
+    if ($preferencesService->get('MediaRichTextCaptions', 'N') == 'N') {
+        $caption = MetaDbObject::htmlFilter($caption);
```

Comment:
My suggestion is to merge it once all clients are upgraded to 4.4.4. Then we can merge it and update clients on the, the clients who use a lot of images should then be verified if they don't already have a logic in their templates that will fix this and possibly conflict with this fix.